nilearn/maskers/tests/test_nifti_maps_masker.py

```diff
@@ -142,9 +124,7 @@ def test_nifti_maps_masker_fit(n_regions, img_maps):
     assert masker.n_elements_ == n_regions
 
 
-def test_nifti_maps_masker_errors(
-    length, n_regions, affine_eye, shape_3d_default, img_maps
-):
+def test_nifti_maps_masker_errors():
```

Comment:
This one seems unrelated. But do we still need this test? I thought this is already there in L196 in class_inspect.py?

Reply:
still needed because this masker cannot accept None as in input at fit time